src/estimation-kdl/CMakeLists.txt

```diff
@@ -29,6 +29,10 @@ target_link_libraries(${libraryname} idyntree-core idyntree-kdl idyntree-sensors
 get_property(IDYNTREE_TREE_INCLUDE_DIRS GLOBAL PROPERTY IDYNTREE_TREE_INCLUDE_DIRS)
 target_include_directories(${libraryname} BEFORE PRIVATE ${IDYNTREE_TREE_INCLUDE_DIRS})
 
+if((${CMAKE_CXX_COMPILER_ID} MATCHES "GNU") OR (${CMAKE_CXX_COMPILER_ID} MATCHES "Clang"))
+    target_compile_options(${libraryname} PRIVATE "-Wno-deprecated")
```

Comment:
Since the next Debian stable (Stretch) [will be released next week](https://release.debian.org/), and Debian is the only distribution with CMake < 3.5, does it make sense to keep supporting CMake 3.0?